README.md

```diff
@@ -70,7 +70,7 @@ Example:
 ```lua
 -- general
 lvim.format_on_save = true
-lvim.colorscheme = "onedarker"
+lvim.colorscheme = "tokyonight"
```

Comment:
sorry! I was out-of-sync with your changes, but this should fix it
```lua
    disable = not vim.startswith(lvim.colorscheme, "tokyonight"),
```

Reply:
that said I vote for keeping all the themes (or at least tokyonight) installed, and only control the setup function using `lvim.colorscheme`, it would save a lot of confusion and headaches with packer

Reply:
yeah i think we should keep all the tokyonight themes as well,
about the `lvim.colorsheme` thingy

do you mean using `lvim.colorscheme.options` instead of `lvim.builtin.theme.options` ?
or ?

Reply:
> yeah, so just remove that `disable` then

oh, I thought since we could disable the previous theme (one-darker), we should let users disable this one as well

Reply:
> oh, I thought since we could disable the previous theme (one-darker), we should let users disable this one as well

the problem with one-darker is that it was very "sticky", as in it was the default as long as it existed, hopefully this won't be the case anymore with tokyonight :no_mouth: 

---

btw, don't forget to remove [`*_document_highlight`](https://github.com/lunarvim/lunarvim/blob/d7c6951463aad9eb80dcc906e51d198882a558e9/lua/lvim/lsp/utils.lua#L84-L115)

Reply:
> btw, don't forget to remove [`*_document_highlight`](https://github.com/lunarvim/lunarvim/blob/d7c6951463aad9eb80dcc906e51d198882a558e9/lua/lvim/lsp/utils.lua#L84-L115)

wait why 👀 ?